teos/src/main.rs

```diff
@@ -103,18 +103,29 @@ async fn main() {
         .init()
         .unwrap();
 
-    if is_default {
-        log::info!("Loading default configuration")
-    } else {
-        log::info!("Loading configuration from file")
-    }
-
     // Create network dir
     let path_network = path.join(conf.btc_network.clone());
     fs::create_dir_all(&path_network).unwrap_or_else(|e| {
         eprintln!("Cannot create network dir: {e:?}");
         std::process::exit(1);
     });
+
+    // Log default data dir
+    log::info!("Default data directory: {:?}", &path);
+
+    // Log datadir path
+    log::info!("Using data directory: {:?}", &path_network);
+
+    // Log config file path based on whether the config file is found or not
+    if is_default {
+        log::info!("Config file: {:?} (not found, skipping)", &conf_file_path);
+    } else {
+        log::info!("Config file: {:?}", &conf_file_path);
+    }
+
+    // Logging the non-default options
+    conf.log_non_default_options();
```

Comment:
Let's move `conf.log_non_default_options();` inside the else branch as they are logically connected.

Reply:
> Let's move `conf.log_non_default_options();` inside the else branch as they are logically connected.

okay.